selftests/unit/test_job.py

```diff
@@ -0,0 +1,21 @@
+import argparse
+import sys
+if sys.version_info[:2] == (2, 6):
+    import unittest2 as unittest
+else:
+    import unittest
+
+from avocado.core import job
+
+
+class JobTest(unittest.TestCase):
+
+    def test_job_empty_suite(self):
+        args = argparse.Namespace()
+        empty_job = job.Job(args)
+        self.assertIsNone(empty_job.test_suite)
+
+    def test_job_empty_has_id(self):
+        args = argparse.Namespace()
+        empty_job = job.Job(args)
+        self.assertIsNotNone(empty_job.unique_id)
```

Comment:
I expect this test to grow. If you keep splitting each step into a different test, we'll end-up with thousands of tests. I know it's all about finding the right balance, but having one-very-basic-test check looks too much. This is not a strong opinion, I just mentioned my concern about it not being too much useful.